bricks/advanced/src/pdf-viewer/index.tsx

```diff
@@ -76,7 +76,8 @@ export function PdfViewerComponent({
   });
 
   return (
-    <Worker workerUrl="https://unpkg.com/pdfjs-dist@3.4.120/build/pdf.worker.min.js">
+    // See: https://react-pdf-viewer.dev/examples/compile-and-set-the-worker-source-with-webpack/
+    <Worker workerUrl="bricks/advanced/dist/pdf.worker.min.js">
```

Comment:
构件包目录可能随着环境变化不同，比如现在都有 `sa-static` 的前缀。这里可以统一使用 __webpack_public_path__ 来获得运行时目录，参考其他地方对构件包内的文件引用：

<img width="594" alt="image" src="https://github.com/user-attachments/assets/4bfa42ec-3ad3-4126-ba2a-b9c594a59cee">


```suggestion
    <Worker workerUrl={`${process.env.NODE_ENV === "test" ? "" : __webpack_public_path__}pdf.worker.min.js`}>
```